.appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 3.3.0.{build}
+
+skip_branch_with_pr: true
+os: Visual Studio 2019
+
+configuration:
+  - Debug
+  - Release
+
+environment:
+  WTG_ANALYZERS_VERSION: 3.3.0
+
+init:
+  - git config --global core.autocrlf true
+
+before_build:
+  - dotnet restore src
```

Comment:
I didn't know you could specify a whole folder like that!

Reply:
If you don't specify a solution/project, it looks for one in the current directory. If you specify a directory, it performs that search in the provided directory.